pinot-core/src/main/java/com/linkedin/pinot/core/operator/filter/SortedInvertedIndexBasedFilterOperator.java

```diff
@@ -49,7 +49,7 @@ public SortedInvertedIndexBasedFilterOperator(PredicateEvaluator predicateEvalua
   }
 
   @Override
-  protected BaseFilterBlock getNextBlock() {
+  public BaseFilterBlock getNextBlock() {
```

Comment:
why are we changing the scope of the method just for testing purpose

Reply:
That's because all the other unit tests for operator are under `com.linkedin.pinot.operator.filter`, while SortedInvertedIndexBasedFilterOperator locates in `com.linkedin.pinot.core.operator.filter`, i.e. in different package.
Do you think we should refactor the location of those unit tests?

Reply:
We should always use nextBlock() instead of getNextBlock().